File: mobile/StudyFragments/app/src/main/java/view/recycler/study/studyfragments/MainActivity.java
package view.recycler.study.studyfragments;

import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        FragmentManager fragmentManager = getSupportFragmentManager();
        FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();

        fragmentTransaction.add(R.id.activity_main_frame_layout_list, new ListPersonFragment());
        fragmentTransaction.commit();

//        FragmentManager fragmentManager = getSupportFragmentManager();
//
//        FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();
//        fragmentTransaction.add(R.id.activity_main_frame_layout_image, new ImageFragment());
//
//        if(findViewById(R.id.activity_main_frame_layout_name) != null)
//            fragmentTransaction.add(R.id.activity_main_frame_layout_name, new ImageNameFragment());
//
////        Way to discover the display orientation ...
////        System.out.printf("Landscape = %s\n", getResources().getBoolean(R.bool.screen_orientation_landscape));
//
//        fragmentTransaction.commit();
    }
}

File: mobile/StudyFragmentsKotlin/app/src/main/java/view/recycler/study/studyfragmentskotlin/MainActivity.kt
package view.recycler.study.studyfragmentskotlin

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.v7.widget.LinearLayoutManager
import android.support.v7.widget.RecyclerView

class MainActivity : AppCompatActivity() {
    lateinit var recyclerView: RecyclerView
    lateinit var layoutManager: RecyclerView.LayoutManager
    lateinit var adapter: RecyclerView.Adapter<*>
    lateinit var listOfPerson: ArrayList<Person>

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        listOfPerson = ArrayList<Person>()
        for( k in 0..10) {
            listOfPerson.add(Person("Jonas", "Freire"))
            listOfPerson.add(Person("Jasom", "Freire"))
            listOfPerson.add(Person("Edson", "Marques"))
            listOfPerson.add(Person("Maria", "Ida"))
            listOfPerson.add(Person("Vinícius", "Alcantara"))
        }

        recyclerView = findViewById(R.id.recycler_view)
        layoutManager = LinearLayoutManager(this)
        adapter = MyAdapter(listOfPerson)

        recyclerView.adapter = adapter
        recyclerView.layoutManager = layoutManager
    }
}
